e2e/upgrade-cephfs.go

```diff
@@ -129,12 +149,45 @@ var _ = Describe("CephFS Upgrade Testing", func() {
 					e2elog.Failf("failed to load application with error %v", err)
 				}
 				app.Namespace = f.UniqueName
-				app.Labels = map[string]string{"app": "upgrade-testing"}
+				app.Labels = map[string]string{"app": "cephfs-upgrade-testing"}
+				pvc.Namespace = f.UniqueName
 				pvc.Spec.Resources.Requests[v1.ResourceStorage] = resource.MustParse(pvcSize)
 				err = createPVCAndApp("", f, pvc, app, deployTimeout)
 				if err != nil {
 					e2elog.Failf("failed to create pvc and application with error %v", err)
 				}
+				opt := metav1.ListOptions{
+					LabelSelector: "app=cephfs-upgrade-testing",
+				}
+				// fetch the path where volume is mounted.
+				mountPath := getMountPath(app)
+				filePath := filepath.Join(mountPath, "testClone")
+
+				// create a test file at the mountPath.
+				cmd := fmt.Sprintf("touch %s", filePath)
```

Comment:
touch does not always help you need to write some data and also you need to run the sync
cc @nixpanic